pyramid/config/views.py

```diff
@@ -568,9 +568,8 @@ def add(self, view, order, accept=None, phash=None):
             else:
                 subset.append((order, view, phash))
                 subset.sort(key=operator.itemgetter(0))
-            accepts = set(self.accepts)
-            accepts.add(accept)
-            self.accepts = list(accepts) # dedupe
```

Comment:
Not sure I agree with the sort logic: in that case the "default" accept will be the lower one (application/foo vs text/bar)

The list/append method have the advantage of defaulting to the first declared view accept